contents/incentivization.rst

```diff
@@ -0,0 +1,189 @@
+
+.. image:: img/swap.svg
+   :alt: Simple SWAP exchange between two nodes 
+   :width: 400
+
+In the above example, A requested data from B, which B then delivers to A (could be a chunk). Assuming a price of 200 units for the chunk, B credits itself in its local database plus 200 at the index for node A, while A, when it finally receives the data from B, debits minus 200 in its local database at the index for node B.
+
+Due to the distributed nature of Swarm nodes, entries in the local databases do not happen simultaneously, and there is no notion of transactions nor confirmations; in normal operation, both nodes account for the message with the correspondent node at the moment of the action (for the sending peer, at the moment of sending, for the receiving peer, at the moment of receiving). 
+
+If a message is being sent, if we would account for it before sending, then a failure in sending would require a complex rollback (or an imbalance). If we send first and then account, then it would not be possible to actually check if there are enough funds. Receiving a message incurs in the same challenge. To address this, Swarm first does a read operation on the database to check if there are enough funds. If positive, the requested operation (send/receive) is performed, and finally if the operation succeeded, the accounting entry is persisted.
+
+Thresholds
+----------
+As described in the [`Incentives system for Swarm <https://swarm-gateways.net/bzz:/swarm.eth/ethersphere/orange-papers/1/sw%5E3.pdf>`_], this accounting system works by allowing the mutual accounting to oscillate in a defined range. This range is delimited by two thresholds:
+* The payment threshold is a number at which if the node goes below that relative to a peer, it should trigger a payment to that peer (a cheque, see below). This is initiated by the debitor node.
+* The disconnect threshold is a number at which if the node goes above that number relative to a peer, it can (it's up to the peer but this is the expected behavior) disconnect the peer. This is monitored by the creditor.
+
+The disconnect threshold amount should be bigger than the payment threshold in absolute terms.
+
+Fraud risks
+-----------
+Of course, the design with individual peer databases means that nodes can alter their database and pretend to have different balances to other nodes. The simplicity of this mutual accounting though effectively significantly limits fraud, as if node A modifies its entry with B, at settlement time, B will verify in its local database that the claim is not matched with its records and simply ignores fraudulent claims. Normal behavior is to disconnect a node in this case. However, we want to make it clear that a node can lose funds up to the disconnect threshold amount due to freeriders.
```

Comment:
```
B will verify in its local database that the claim is not matched with its records and simply ignores fraudulent claims
```
To which code is this referring to?

Reply:
Actually it was meant to express that if the balances don't match, nodes will reject claims of any kind.

Reply:
But what is a "claim" here?

Reply:
I rephrased